README.md

```diff
@@ -1 +1,166 @@
-# Final_capstone_Back-End
\ No newline at end of file
+# 📗 Table of Contents
```

Comment:
_note: I couldn't find an open PR on the frontend to review. For your next review request, please open one. It should contain all the content and point from dev to main. Therefore, I'll put my frontend review comments here:_

## Frontend project

- [ ] The required readme based on the [Microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) is not present on the frontend repo.  Kindly go add it and customize it for your project.

- [ ] The App.test.js file is the default test that comes with CRA.  Kindly delete it if you are not using that test.

- [ ] Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://user-images.githubusercontent.com/57421823/208198821-96e21813-f1a0-45f8-aa6c-e667cb0382ce.png)

- [ ] It was impossible for me to run the frontend app:

![image](https://user-images.githubusercontent.com/57421823/208198953-779bc4fe-f977-4db7-b236-072ce867a9e5.png)

Kindly fix this bug so your app works as intended.

Reason 1: 

You've imported `Provider` twice on index.js.

Solution: Delete either line 3 or 6.

Reason 2:  You have a `BrowserRouter as Router` and another `BrowserRouter` on both App.js and index.js.  You can never have 2 of them.  

![image](https://user-images.githubusercontent.com/57421823/208199660-abee82a7-e5fa-4113-9469-88dfc7217229.png)


Solution:  Eliminate one BrowserRouter call and correct the routes.

![image](https://user-images.githubusercontent.com/57421823/208199292-9f5d61f3-d313-4ee8-a417-9ef85786c9b5.png)

![image](https://user-images.githubusercontent.com/57421823/208199377-7e38391f-ee05-41f6-a286-f90f1e262460.png)

After doing this, I could get into the app, but I couldn't navigate any links, because the routes still have issues.  Kindly correct your routes so your app functions as intended.

_Since I couldn't run the frontend, I wasn't able to test the actual app functionality.   Before your next review request, please make sure that the rest of the functionality is correct._

- [ ] Your frontend app has many linter errors.   Kindly correct them.

That way, your frontend app will work as intended, and will look professional.  💪